ebpf/Makefile.am

```diff
@@ -18,7 +18,7 @@ all: $(BPF_TARGETS)
 
 $(BPF_TARGETS): %.bpf: %.c
 #      From C-code to LLVM-IR format suffix .ll (clang -S -emit-llvm)
-	${CLANG} -Wall $(BPF_CFLAGS) -O2 \
+	${CLANG} -Wall $(BPF_CFLAGS) -O2 -g \
```

Comment:
the -g flag makes bpf object file containing BTF information that the new libbpf can parse for BTF map, otherwise, libbpf will complain missing BTF information from the bpf object file and fail, you can also run bpftool btf dump file <bpf object file> to check the BTF information